tests/conftest.py

```diff
@@ -383,13 +383,13 @@ def empty_config(tmp_path) -> Config:
 
 
 @pytest.fixture
-def profile_factory() -> Profile:
+def profile_factory():
```

Comment:
Still the type you are returning can be specifiey`Callable[Concatenate[str, P], Profile]` using `P = ParamSpec("P")`

I don't know how much it makes sense to have type hints in tests, but it seems also like utils we would like to ship for developer at some point